### ISSUE TITLE
Data is not inserted into database. 

### ISSUE THREAD

**Issue by sreyas** [malikshubham827/get-me-through#5]
vistors table is empty at every time

**Comment by malikshubham827** [comment]
Yes, it is because the addition of the record depends on the business logic which is subjective depending on the use case. You can also see that I have mentioned [here](https://github.com/malikshubham827/get-me-through/blob/master/server/models/visitor.js#L53) to include the separate fields for custom purpose.
It is just a barebone project with _no assumptions_.

I intend to add the CRUD functionality and other things, but I am currently onto something else so wouldn't be able to do it any time soon.
